validator/webui/index.html

```diff
@@ -389,6 +391,7 @@
 
           // The status bar below the editor.
           var statusBar = this.$.statusBar;
+          statusBar.status = 'PASS';  // Start out valid.
```

Comment:
NSO: perhaps we should actually load the file and run the validation since at some point what is currently hardcoded here may no longer be valid.